packages/replay/src/types.ts

```diff
@@ -196,6 +196,39 @@ export interface SampleRates {
   errorSampleRate: number;
 }
 
+export interface ReplayNetworkOptions {
+  /**
+   * Capture request/response details for XHR/Fetch requests that match the given URLs.
+   * The URLs can be strings or regular expressions.
+   * When provided a string, we will match any URL that contains the given string.
+   * You can use a Regex to handle exact matches or more complex matching.
+   *
+   * Only URLs matching these patterns will have bodies & additional headers captured.
+   */
+  networkDetailAllowUrls: (string | RegExp)[];
+
+  /**
+   * If request & response bodies should be captured.
+   * Only applies to URLs matched by `networkDetailAllowUrls`.
+   * Defaults to true.
+   */
+  networkCaptureBodies: boolean;
+
+  /**
+   * Capture the following request headers, in addition to the default ones.
+   * Only applies to URLs matched by `networkDetailAllowUrls`.
+   * Any headers defined here will be captured in addition to the default headers.
```

Comment:
Should we list the default headers here?

Reply:
I'd just document them in the docs, IMHO should be good enough? As this may change, I guess.